js/privatebin.js

```diff
@@ -2225,6 +2221,18 @@ jQuery.PrivateBin = (function($, RawDeflate) {
             return $message.val();
         };
 
+        /**
+         * returns if status is editing
+         *
+         * @name   Editor.isEditing
+         * @function
+         * @return {bool}
+         */
+        me.isEditing = function()
```

Comment:
I noticed that in all cases that you call this you negate the result. Hope you don't mind, but I'll remove the "not" from this functions return all the calls of it and rename it accordingly (i.e. Editor.isHidden), so the logic is a bit clearer.

Reply:
I'm also looking into this just then, I believe this function and its usage introduce some undesired assumption, e.g. attachment is editable only when textarea visible, I propose to change it to use following function instead:
```
       /**
         * returns if attachment dropdown is editable
         * 
         * @name   TopNav.isAttachmentEditable
         * @function
         * @return {bool}
         */
        me.isAttachmentEditable = function()
        {
            return !$attach.hasClass('hidden');
        }
```

Reply:
@elrido Will we still change that last thing @Haocen mentioned?

Reply:
I'm working on a seperate PR right now, sorry for the delay.

Reply:
Sorry, I didn't keep up with reading the updates. Let's address this in that other PR then. Sorry again for the hassle.